README.md

```diff
@@ -175,7 +175,7 @@ Returns a [result] or `null`.
 ## explorer.load()
 
 ```js
-explorer.load([loadPath]).then(result => {..})
+explorer.load(loadPath).then(result => {..})
```

Comment:
If we want to remove the brackets indicating that it's optional (I'm fine with that), let's do it in every place — so also in `search` and `searchSync`.

Reply:
it is optional in search though.  it's mandatory in load

Reply:
Ooooo, right 👍